src/tools/scripts/deploy.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# exit when any command fails instead of trying to continue on
+set -e
+
+echo "Update local production branch"
+git checkout production
```

Comment:
Not sure why, but I get nervous running git commands automatically. Should we assume the deployer has already manually updated the branch and maybe exit early from the script if it detects unmerged changes?

Reply:
And that’s why I check before pushing it (which I see you asked about later).

Wonder how to check for unmerged changes. Would be nice if there’s a simple way to check that. Will have a google.

Reply:
I've added a check to see if it's a clean branch (with option to continue even if not) and also exit on first error so think this is safe to automate.